nautobot/extras/api/views.py

```diff
@@ -375,7 +375,7 @@ class GitRepositoryViewSet(NautobotModelViewSet):
     filterset_class = filters.GitRepositoryFilterSet
 
     @extend_schema(methods=["post"], request=serializers.GitRepositorySerializer)
-    @action(detail=True, methods=["post"])
+    @action(detail=True, methods=["post"], permission_classes=[IsAuthenticated])
```

Comment:
Would be good to add an explanatory comment about the permission_classes, I think?